### PR TITLE
Added description for the type of files

### PR DESCRIPTION
**Pull request by derhallim**
Added description for type of files to be available for upload.
Related to issue #2663

### CODE REVIEW

**Review by derhallim — COMMENTED** [review]
Removed extra text.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/organization-assets-library.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If your organization needs to store and manage files for all your users to use,
 
     ![Selecting an image to add to a SharePoint page](media/image-library.png)
 
-- **Office templates**. When a user selects to create a new PowerPoint presentation (from PowerPoint for the web or the PowerPoint desktop app), the user can select the tab for your organization to see the templates. (To use this feature on PowerPoint for the web, users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a file from a template, go to the PowerPoint start page > Office Template Library.)
+- **Office templates**. When a user selects to create a new PowerPoint presentation (from PowerPoint for the web or the PowerPoint desktop app), the user can select the tab for your organization to see the templates. (To use this feature on PowerPoint for the web, users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a file from a template, go to the PowerPoint start page > Office Template Library). Also note that the Office files to be uploaded to the template libraries must be in a template format, for example, dotx for Word, potx for PowerPoint and xltx for Excel.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/organization-assets-library.md:33]:
```suggestion
- **Office templates**. When a user selects to create a new PowerPoint presentation (from PowerPoint for the web or the PowerPoint desktop app), the user can select the tab for your organization to see the templates. (To use this feature on PowerPoint for the web, users need to be assigned a license to Office 365 E3 or E5. The templates aren't available from the New menu. To create a file from a template, go to the PowerPoint start page > **Office Template Library**.) Also note that the Office files to be uploaded to the template libraries must be in a template format, for example, dotx for Word, potx for PowerPoint, and xltx for Excel. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**